homeassistant/components/asuswrt/sensor.py

```diff
@@ -156,6 +156,26 @@ class AsusWrtSensorEntityDescription(SensorEntityDescription):
         entity_registry_enabled_default=False,
         suggested_display_precision=1,
     ),
+    AsusWrtSensorEntityDescription(
```

Comment:
> Ok, I'll stop at this point. Please reduce this PR to the core goal you want to achieve.
>
> PR should be small and solve a single thing. This adds translations, unique ID migrations, new sensors, and more.
>
> ../Frenck


As Frenck already has written, this PR is too big. Please move all new sensors in a follow-up PR.

Reply:
Ok, I already removed new sensors, but I didn't understand that I have to create a follow up PR. I'll do, thanks

Reply:
The sensors are still part of this PR and therefore not removed.

Reply:
Oh, yes, this was just some additional type of the same "cluster" (temperature), but I can remove also this.
But do you think it is better to just remove this sensors from this PR or create a new one? I also made a lot of changes related to the tests, should I create a separate PR to review test structure before?

Thanks.

Reply:
Ok, I removed this additional sensors. 
what complicates this PR is also the part related to the migration to the new protocol. If you think is better to move this on a separate PR, I can remove from here.

Reply:
The PR should as small as possible.
Tests for the new protocol can stay in this PR. If you improved the test coverage for example sensor, than those tests should go into a new PR

Reply:
Mainly I reorganize the test, coverage is quite unchanged. But may be is better to also move the test reorganization in a separate PR. 
I put this in draft for now, let me see how I can do this as small as possible.
Can i just have an opinion about my last question? Do you think that automatic migration to http protocol should be part of this PR or may stay separate in a follow up PR?

Reply:
Is the migration to HTTP required? Is ssh and telnet not working anymore?
What are the advantages and disadvantages of migrating to HTTP(S)?
Please update the PR description accordingly.

Example:
The user is currently using SSH with the password. 
HTTPS is not set up on the router, only HTTP.

With this PR, the user is migrated automatically to HTTP without interaction. The user still thinks he uses SSH (which is encrypted) but in fact, the integration uses unencrypted plain HTTP.
This is a no-go in my opinion and a security risk too.

Reply:
Yes, I fully agree with you. I will update description, but I think that automatic migration is not a good approach. But the user should be notified about new options. So may be we can create a repair to provide this information that the user can just ignore should be the best solution.

Reply:
Please answer me the above questions:
> Is the migration to HTTP required? Is ssh and telnet not working anymore?
> What are the advantages and disadvantages of migrating to HTTP(S)?

Repair issues are here to bring a problem/issue with the integration to the user's attention. Not to inform the user about some other supported protocols.

Especially imo SSH is encrypted and should be recommended over plain http

Reply:
Ok, thanks for suggestion. I will think about a different approach for this and meantime I will create some PR related only to tests.

Reply:
To answer to your  questions:

> Is the migration to HTTP required? 

There are many issue with previous library using last AsusWRT firmware. Migration is not required because integration still work with old library, but how the user will be informed about new protocol that potentially solve the issue is available 

> Is ssh and telnet not working anymore?
ssh and telnet are still working

> What are the advantages and disadvantages of migrating to HTTP(S)

I don't see any disadvantages, but old firmware are not compatible with HTTP(S). Of course replace SSH with HTTP is a security risk. But SSH to HTTPS may be not. Probably we should never migrate automatically to new protocol, just invoke a `reconfiguration` allowing the user to make the proper choice